lib/transports/multi-key-rabbit.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const Joi = require('@hapi/joi');
+
+const optionsSchema = Joi.object({
+    exchangeType: Joi.string().allow(['default', 'topic']).default('topic'),
+    exchangeName: Joi.string().required()
+});
+
+class RabbitTransport {
```

Comment:
nitpick: we might want to name classes aligned with their filename/function
```suggestion
class MultiKeyRabbitTransport {
```